vita3k/config/src/config.cpp

```diff
@@ -289,7 +289,7 @@ ExitCode init_config(Config &cfg, int argc, char **argv, const Root &root_paths)
         po::options_description config_desc("Configuration");
         config_desc.add_options()
             ("archive-log,A", po::bool_switch(&command_line.archive_log), "Makes a duplicate of the log file with TITLE_ID and Game ID as title")
-            ("backend-renderer,B", po::value(&command_line.backend_renderer), "Renderer backend to use, either \"OpenGL\" or \"Vulkan\"")
+            ("backend-renderer,B", po::value<std::string>(&command_line.backend_renderer), "Renderer backend to use, either \"OpenGL\" or \"Vulkan\"")
```

Comment:
This ought to be redundant too. Can you remove the `<std::string>`? C++ should add it for you.

Reply:
ah, i have set it because other string here have this here ^^

Reply:
Okay 😄 should be good without it if you want to take the time to remove it.

Reply:
ah oki :D